lib/bin/purge-forms.js

```diff
@@ -16,7 +16,8 @@ const { purgeForms } = require('../task/purge');
 const cli = require('cli');
 const cliArgs = {
   force: [ 'f', 'Force any soft-deleted form to be purged right away.', 'bool', false ],
-  formId: [ 'i', 'Purge a specific form based on its id.', 'int' ]
+  formId: [ 'i', 'Purge a specific form based on its id.', 'int' ],
+  projectId: [ 'p', 'Restrict purging to a specific project.', 'int' ],
```

Comment:
`options.projectId` needs to be passed into `purgeForms` a few lines down in this same file!

Reply:
ooops, thanks!